libs/sdk-core/src/persist/sync.rs

```diff
@@ -170,7 +170,8 @@ impl SqliteStorage {
           lnurl_metadata,
           lnurl_withdraw_endpoint,
           attempted_amount_msat,
-          attempted_error
+          attempted_error,
+          external_metadata
```

Comment:
As a side note, I was wondering whether or not it would would be useful to keep the `external_metadata` in the backup, as its usefulness could vary based on what the implementation is like. Given the small size, perhaps it is fine for now?

Reply:
I agree.  It only I think it is fine but also it seems mandatory since we want this metadata to sync accross apps/devices